### PR TITLE
Fixed #255: Travis failing; and adding myself to the credits

### LINKED ISSUE #255: Travis is failing
Travis has been failing, and it has several errors. I am working on this, Cheers 😄 !

### PR DESCRIPTION
**Pull request by Heredroky**
With this PR I solved #255 and I also added myself to the credits page. I hope you like!

You can check my travis build [here](https://travis-ci.org/Heredroky/flappy-svg/builds/191514948)!

Cheers! 😄

### CODE REVIEW

File: main_menu.html [hunk 1]
```diff
@@ -75,7 +75,7 @@ <h2 class="list-group-item active" id="mainmenu">MAIN MENU</h3>
 			</a>
 		</div> 
     <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
-    <script src="style/bootstrap/js/jquery.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
```
Comment (niccokunzmann) [inline main_menu.html]: This loads content from the web. Please download the jquery file and use a local link.

**Comment by Heredroky** [comment]
@niccokunzmann Check it please, I have just done it 😄

**Comment by Abhi2424shek** [comment]
@Heredroky nice work merging... next time I want to see you squashing the commits pls.

**Comment by Heredroky** [comment]
@Abhi2424shek I didn't squash them, because each commit added a different functionality.

**Comment by Heredroky** [comment]
@Abhi2424shek It is to have a better order...

**Comment by Abhi2424shek** [comment]
@Heredroky but we squash it when merging it, I nor most of the maintainers wont enforce you to squash but I meant that you should learn squashing this competition should also be used to improve your knowledge base :)

**Comment by Heredroky** [comment]
@Abhi2424shek I already know how to squash...

**Comment by Abhi2424shek** [comment]
i just mentioned as I never knew that @Heredroky anyways,
Happy coding :)

**Pull request merged**